synapse/cortex.py

```diff
@@ -116,9 +118,10 @@ def getCoreInfo(self):
         Adds a trigger to the cortex
         '''
         username = None if self.user is None else self.user.name
-        self.cell.triggers.add(username, condition, query, info=info)
+        iden = self.cell.triggers.add(username, condition, query, info=info)
+        return iden
 
-    def _trig_auth_check(self, triguser):
+    def _auth_check(self, triguser):
```

Comment:
argument is `triguser` but we may not be using this for triggers

Reply:
ok